internal/pkg/config/v1/api_key_pair.go

```diff
@@ -2,6 +2,6 @@ package v1
 
 // APIKeyPair holds an API Key and Secret.
 type APIKeyPair struct {
-	Key    string `json:"api_key" hcl:"api_key"`
```

Comment:
We're not using HCL tags anywhere as far as I can tell. Maybe at one point we wanted the configuration file to have HCL format, but I think that's a bad idea because people are much more familiar with JSON than HCL.